### DISCUSSION TITLE
Number and red cross

### DISCUSSION THREAD

**Discussion by Moberaptor** [lxqt/lxqt#1981] [Q&A]
Hello!
I'm using QTerminal on Kale. Could you please help me... I found a strange number and red cross in my terminal. What is that?
![index](https://user-images.githubusercontent.com/68503182/115277698-81a33a80-a13c-11eb-94a4-cedc86b69d95.jpg)
I can't find what is it...
Could you explain it please? Is it normal?
Thank you!

**Comment by tsujan** [comment]
Please attach your bash prompt.
